pkg/nodebootstrap/windows.go

```diff
@@ -35,8 +35,8 @@ func (b *Windows) UserData() (string, error) {
 		`<powershell>
 [string]$EKSBootstrapScriptFile = "$env:ProgramFiles\Amazon\EKS\Start-EKSBootstrap.ps1"`,
 	}
-
-	bootstrapCommands = append(bootstrapCommands, b.ng.PreBootstrapCommands...)
+	ng := (b.np).BaseNodeGroup()
+	bootstrapCommands = append(bootstrapCommands, ng.PreBootstrapCommands...)
```

Comment:
Hi @cPu1, is PreBootstrapCommands a valid use case for MNG?

Reply:
It's supported for self-managed Windows, so I think we should support it.